pytest.ini

```diff
@@ -0,0 +1,4 @@
+[tool:pytest]
```

Comment:
Why do you use it? Why you can not use standard tests?

Reply:
done